src/view/com/util/List.web.tsx

```diff
@@ -445,7 +450,9 @@ let Row = function RowImpl<ItemT>({
   }
 
   return (
-    <View style={styles.row} ref={rowRef}>
+    <View
+      style={disableContentVisibility ? undefined : styles.row}
```

Comment:
This is a bit confusing since it's easy to add something to `styles.row` without realizing it's conditional on this prop. Something in naming needs to bind them better together.